vscode/src/completions/latency.ts

```diff
@@ -0,0 +1,54 @@
+import { logDebug } from '../log'
+
+import { LastInlineCompletionCandidate } from './get-inline-completions'
+
+const defaultLatency = {
+    baseline: 600,
```

Comment:
```suggestion
    baseline: 400,
```
<img width="883" alt="image" src="https://github.com/sourcegraph/cody/assets/11155207/d969e3ee-9063-4cf1-a2ec-b7b7f94b0a16">

Reply:
we have updated the default behavior at the end, but want to thank you again for always sharing your thoughts on how we can improve and fix the issues  <3